tests/deploy_test.sh

```diff
@@ -153,10 +153,10 @@ function test_kernel_deploy()
 
   # Test some invalid parameters
   ID=9
-  output=$(kernel_deploy test_mode --rem 192.168.0.10:1287)
+  output=$(kernel_deploy test_mode --remtoe 192.168.0.10:1287)
```

Comment:
it looks like that we are matching "partial" strings, and I suppose this is why you changed this test (--rem does not fail as previously expected). Can we make getopt or the case condition only accept the exact match?

Reply:
Exactly! I had commented about it in the commit description. It seems there is no way to make getopt change this behavior, and we cannot easily change this in the case condition, since getopt detects `--rem` as being `--remote` and outputs `--remote` to be parsed, even if the user supplied `--rem`. Since there is no ambiguity (getopt makes sure of that) I propose that we accept this behavior.